ompi/datatype/ompi_datatype.h

```diff
@@ -49,6 +49,7 @@ BEGIN_C_DECLS
 #define OMPI_DATATYPE_FLAG_ANALYZED      0x0400
 #define OMPI_DATATYPE_FLAG_MONOTONIC     0x0800
 /* Keep trace of the type of the predefined datatypes */
+#define OMPI_DATATYPE_FLAG_DATA_BOOL     0x0000 /* TODO: what to do here? */
```

Comment:
I need help here. I've run out of space for an extra bit to distinguish boolean types. Can I shift everything bit up to make room?